### PR TITLE
Updates and bug fixes

### LINKED ISSUE #361: ad unit: can't modify existing tag when cloning
**Describe the bug**
When cloning an ad unit, existing targeting units cannot be modified

**To Reproduce**
1. Clone an ad unit
2. try to modify the existing targeting tags
3. dropdown shows up but you can't touch the text

### LINKED ISSUE #362: Campaign statistics: ad unit breakdown by impressions/clicks/CTR
**Is your feature request related to a problem? Please describe.**
If you're an advertiser, it's impossible to tell which ad units are performing the best.

Putting this in v4.1 since we've essentially missed it/forgotten it

**Describe the solution you'd like**
When you open a campaign, the ad unit table should also contain impressions/clicks/CTR

**Describe alternatives you've considered**
If advanced stats do not contain campaignId->unitId breakdowns, just show a table of the best performing units overall in the dashboard, or integrate it in the Ad Units (/dashboard/advertiser/units) page

### LINKED ISSUE #369: Close campaign UX issues
**Describe the bug**
Closing a campaign is quite confusing for a few reasons

* the button is not noticable; I suggest making it solid (contained) and bigger; also, clicking the media should open the campaign detail page (the same page that "view" opens)
* the status does not update immediately after pressing "close" 
* the Close button keeps showing after the campaign is in a Closed status
* it takes a few mins for the funds to show up, so a toast "Your campaign has been closed. The remaining funds will be returned to you in a few minutes." would be nice

slightly related:
* before a campaign is done, show the Report button but with a tooltip that says "Report not available until the campaign is completed"; otherwise the user does not know why the button exists in some cases and not in others

SUMMARY TODO (Rangel):
- [x] make button contained
- [x] make button bigger
- [x] media should open campaign detail page
- [x] update status immediately after pressing close
- [x] notify user "Your campaign has been closed. The remaining funds will be returned to you in a few minutes."
- [x] show receipt button when campaign not completed with tooltip "Report not available until the campaign is completed"

### LINKED ISSUE #371: Campaign statistics: show CTR and name on "campaigns" tab
**Is your feature request related to a problem? Please describe.**
The campaigns tab does not show CTR and name

**Describe the solution you'd like**
- [x] Show the campaign name.
- [x] The table also has clicks and impressions, so it can easily show CTR as well

### LINKED ISSUE #382: Update Changelog link in platform
Please change link for the changelog  from https://github.com/AdExNetwork/adex-platform/blob/master/CHANGELOG.md to https://github.com/AdExNetwork/adex-platform/blob/development/CHANGELOG.md. 

Screenshot 2020-03-04 at 17 06 16

### LINKED ISSUE #383: Add link to ToS in the platform
Please add a link that reads "Terms and Conditions" (https://www.adex.network/tos) in the lower left corner of the platform screen: 

Screenshot 2020-03-04 at 18 06 49

### PR DESCRIPTION
**Pull request by rori4**
- Add link to ToS in the platform (closes #383)
- Update Changelog link in platform (closes #382)
- Close campaign UX issues (closes #369)
- Campaign statistics: show CTR and name on "campaigns" tab (closes #371)
- ad unit: can't modify existing tag when cloning (closes #361)
- Campaign statistics: ad unit breakdown by impressions/clicks/CTR (closes #362)

### CODE REVIEW

File: src/components/dashboard/containers/Tables/CampaignsTable.js [hunk 1]
```diff
@@ -112,7 +124,7 @@ const getCols = ({
 	},
 	{
 		name: 'fundsDistributedRatio',
-		label: t('PROP_DISTRIBUTED'),
+		label: t('PROP_DISTRIBUTED', { args: ['%'] }),
```
Comment (ivopaunov) [inline src/components/dashboard/containers/Tables/CampaignsTable.js]: If the label has `%` remove it from the `customBodyRender`

File: src/components/dashboard/containers/Tables/CampaignsTable.js [hunk 2]
```diff
@@ -147,6 +159,18 @@ const getCols = ({
 			}),
 		},
 	},
+	{
+		name: 'ctr',
+		label: t('LABEL_CTR'),
```
Comment (ivopaunov) [inline src/components/dashboard/containers/Tables/CampaignsTable.js:163]: Be consistent - put `%` in the label or in the value

File: src/components/dashboard/containers/Campaign/Campaign.js [hunk 3]
```diff
@@ -137,7 +141,7 @@ export class Campaign extends Component {
 							</Grid>
 						)}
 						{tabIndex === 1 && (
-							<AdUnitsTable items={units} campaignUnits noClone />
+							<AdUnitsTable campaignId={campaign.id} noClone />
```
Comment (ivopaunov) [inline src/components/dashboard/containers/Campaign/Campaign.js:142]: remove `const units = item.spec.adUnits` it is not used any more

**Review by ivopaunov — CHANGES_REQUESTED** [review]

File: src/components/dashboard/containers/Tables/CampaignsTable.js (hunk 1 above)
Comment (rori4) [reply src/components/dashboard/containers/Tables/CampaignsTable.js]: The translation had a placeholder so this is why I have added it. The translation was displayed as 'Served 0'

**Pull request merged**